Tests/Request/RequestBodyParamConverter20Test.php

```diff
@@ -182,8 +190,8 @@ public function testApplyWithSerializerContextOptionsForSymfonySerializer()
     {
         $this->serializer = $this->getMock('Symfony\Component\Serializer\SerializerInterface', array('serialize', 'deserialize'));
         $this->converter = new RequestBodyParamConverter20($this->serializer);
+        $this->fillContainer($this->converter);
         $requestBody = '{"name": "Post 1", "body": "This is a blog post"}';
-
         $options = array(
```

Comment:
I can update the ``AbstractRequestBodyConverter::configureDeserializationContext()`` method like this :
```php
    /**
     * @param ContextInterface $context
     * @param array            $options
     *
     * @return ContextInterface
     */
    protected function configureDeserializationContext(ContextInterface $context, array $options)
    {
        foreach($options as $key => $value) {
            if ($key == 'group' && $context instanceof GroupableContextInterface) {
                $context->addGroups($options['groups']);
            } elseif ($key == 'version' && $context instanceof VersionableContextInterface) {
                $context->setVersion($options['version']);
            } elseif ($key == 'maxDepth' && $context instanceof MaxDepthContextInterface) {
                $context->setMaxDepth($options['maxDepth']);
            } elseif ($key == 'serializeNull' && $context instanceof SerializeNullContextInterface) {
                $context->setMaxDepth($options['maxDepth']);
            } else {
                $context->setAttribute($key, $value);
            }
        }

        return $context;
    }
```
Is it ok for you ?